virt/live_migration/virt-configuring-live-migration.adoc

```diff
@@ -0,0 +1,36 @@
+:_content-type: ASSEMBLY
+[id="virt-configuring-live-migration"]
+= Configuring live migration
+include::_attributes/common-attributes.adoc[]
+:context: virt-configuring-live-migration
+
+toc::[]
+
+You can configure live migration parameters and create live migration policies for your cluster.
+
+You can also configure eviction strategies for virtual machines (VMs) created from custom templates.
+
+Live migration settings::
+You can configure live migration limits and timeouts by using the xref:../../virt/live_migration/virt-configuring-live-migration.adoc#virt-configuring-live-migration-limits_virt-configuring-live-migration[command line] to ensure that the migration processes do not overwhelm the cluster.
++
+You can configure the maximum number of migrations per node or cluster or a dedicated live migration network by using the xref:../../virt/getting_started/virt-web-console-overview.adoc#overview-settings-cluster_virt-web-console-overview[web console].
+
+Live migration policies::
+Live migration policies define different migration configurations for specified groups of VMs. You can create live migration policies by using the xref:../../virt/getting_started/virt-web-console-overview.adoc#migrationpolicies-page_virt-web-console-overview[web console] or the xref:../../virt/live_migration/virt-configuring-live-migration.adoc#virt-configuring-a-live-migration-policy_virt-configuring-live-migration[command line].
++
+--
+:FeatureName: Live migration policy
+include::snippets/technology-preview.adoc[]
```

Comment:
LiveMigration Policy is GA now. https://issues.redhat.com/browse/CNV-19056
So we should drop this now.